src/Components/Form.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+import "./Form.css";
+
+const Form: React.FC = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [emailalert, setEmailAlert] = useState("");
+  const [passwordalert, setPasswordAlert] = useState("");
+
+  const validateForm = () => {
```

Comment:
why you don't use an OOP way to write this code !!

Reply:
hello Elias
I used the OOP way. 
please check my code
thank you

Reply:
@ameer1004 **if you are sure that you answer my question you need to resolve this** because you still have these issues open, if you think that your resolve is not good you can let her open until I write for you it's okay

Reply:
Hi Elias 
I resolved your question and updated my code using the OOP way. 
I think my code is working and it's good now.